src/concrete/ml/deployment/fhe_client_server.py

```diff
@@ -124,37 +124,68 @@ def load(self):
 
         self.server = fhe.Server.load(Path(self.path_dir).joinpath("server.zip"))
 
+    # We should make 'serialized_encrypted_quantized_data' handle unpacked inputs, as Concrete does,
+    # instead of tuples
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4477
+    # We should also rename the input arguments to remove the `serialized` part, as we now accept
+    # both serialized and deserialized input values
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4476
     def run(
```

Comment:
we now allow serialized and deserialized inputs (avoids having to deser + ser at each server call with composition)